### PR TITLE
Svfg-rw Build Fix

### PR DESCRIPTION
**Pull request by Jeffrey-J-Ma**
Testing if build passes

### CODE REVIEW

File: lib/Util/Options.cpp [hunk 1]
```diff
@@ -471,6 +471,17 @@ llvm::cl::opt<bool> Options::OPTSVFG(
     llvm::cl::desc("Optimize SVFG to eliminate formal-in and actual-out")
 );
 
+const llvm::cl::opt<std::string> Options::WriteSVFG(
+    "write-svfg",  
+    llvm::cl::init(""),
+    llvm::cl::desc("Write Andersen's analysis results to a file")
```
Comment (yuleisui) [inline lib/Util/Options.cpp:477]: change the description

File: lib/Util/Options.cpp [hunk 2]
```diff
@@ -471,6 +471,17 @@ llvm::cl::opt<bool> Options::OPTSVFG(
...
+);
+
+const llvm::cl::opt<std::string> Options::ReadSVFG(
+    "read-svfg",  
+    llvm::cl::init(""),
+    llvm::cl::desc("Read Andersen's analysis results from a file")
```
Comment (yuleisui) [inline lib/Util/Options.cpp:483]: change the description

File: lib/WPA/FlowSensitiveStat.cpp [hunk 3]
```diff
@@ -349,8 +349,9 @@ void FlowSensitiveStat::statPtsSize()
         if (size > _MaxTopLvlPtsSize)	_MaxTopLvlPtsSize = size;
     }
 
-    if (totalValidTopLvlPointers != 0)
+    if (totalValidTopLvlPointers != 0) {
```
Comment (yuleisui) [inline lib/WPA/FlowSensitiveStat.cpp:352]: roll back

File: lib/Graphs/SVFG.cpp [hunk 4]
```diff
@@ -36,7 +36,7 @@
 #include "Util/Options.h"
 #include "MemoryModel/PointerAnalysisImpl.h"
 #include <fstream>
-
+#include "Util/Options.h"
```
Comment (yuleisui) [inline lib/Graphs/SVFG.cpp:39]: move these two headers to SVFGWriteRead.cpp

move `writeToFile readFile` to SVFGWriteRead.cpp

**Comment by Jeffrey-J-Ma** [comment]
will create new pull request with a cleaned up branch

**Pull request closed without merging**